src/js/src/helper.cpp

```diff
@@ -337,6 +338,28 @@ ov::Tensor js_to_cpp<ov::Tensor>(const Napi::Env& env, const Napi::Value& value)
     return *tensor_ptr;
 }
 
+template <>
+std::vector<ov::Tensor> js_to_cpp<std::vector<ov::Tensor>>(const Napi::Env& env, const Napi::Value& value) {
+    std::vector<ov::Tensor> tensors;
+    if (value.IsUndefined() || value.IsNull()) {
+        return tensors;
+    }
+    if (value.IsArray()) {
+        auto array = value.As<Napi::Array>();
+        size_t length = array.Length();
+        tensors.reserve(length);
+        for (uint32_t i = 0; i < length; ++i) {
+            tensors.push_back(js_to_cpp<ov::Tensor>(env, array[i]));
+        }
+    } else if (value.IsObject()) {
+        // Single tensor
+        tensors.push_back(js_to_cpp<ov::Tensor>(env, value));
```

Comment:
Does it mean that API like `images=ov::Tensor` is supported?
I would force users to follow `images=std::vector<ov::Tensor>` API for `images` argument, e.g. in JS `generate(..., { images: [image_tensor], ...})`.